docs/dev/annual_updates.rst

```diff
@@ -156,57 +142,140 @@ B. FERC Form 1
-``fuel_ferc1`` table are reported as freeform strings and need to be converted to simple
-categorical values to be useful. If the new year of data contains strings that have
-never been encountered before, they need to be added to the string cleaning dictionaries
-defined in :mod:`pudl.transform.ferc1`. The ``devtools/ferc1/ferc1-new-year.ipynb``
-notebook and :func:`pudl.helpers.find_new_ferc1_strings` will help with this process.
-Every string observed in these fileds should ultimately be mapped to one of the defined
-categories.
-
-5. Update the PUDL DB schema
-----------------------------
-**5.1)** If new columns or tables have been added, you will need to update the
-PUDL DB schema, defining column types, giving them meaningful descriptions, applying
-appropriate ENUM constraints, etc. This happens in the :mod:`pudl.metadata` subpackage.
-Otherwise when the system tries to write dataframes into SQLite, it will fail.
+to using columns. In the Pre-2021 data (from the DBF files), rows are identified by
+``row_number``, and the row number that corresponds to a given variable changes from
+year to year. We cataloged this correspondence in the FERC 1 row maps, a collection of
+CSV files stored under ``src/pudl/package_data/ferc1/row_maps`` and organized it by
+original FERC 1 DB table name.
+
+The FERC 1 data from 2021 onwards (XBRL files) is organized into a taxonomy that keeps
+it relatively consistent year to year. We keep track of the connection between DBF and
+XBRL tables in the ``src/pudl/package_data/ferc1/dbf_to_xbrl_tables.csv`` and
+``src/pudl/package_data/ferc1/dbf_to_xbrl.csv`` files.
+
+**4.B.1)** If there are any new tables or reason to believe that the xbrl taxonomy has
+changed, revisit the ``dbf_to_xbrl_tables.csv`` and ``dbf_to_xbrl.csv`` files and map
+the tables and records to one another.
+
+.. note::
 
-**5.2)** Differentiate between columns which should be harvested from the transformed
+    **How to use the mapping spreadsheets:**
+
+    The ``dbf_to_xbrl.csv`` maps row numbers from the DBF data with taxonomy factoids
+    from the XBRL data therefore allowing us to merge the data into one continuous
+    timeseries. The ``row_literal`` column is the DBF label for the ``row_number`` in
+    question. This ``row_literal`` must be mapped to an ``xbrl_factoid`` from the XBRL
+    data. These ``xbrl_factoid`` entires are the value columns from the raw XBRL data.
+
+    Look at the ``row_literal`` values for a given table and see which XBRL columns they
+    coorespond to. It's helpful to
+    `view the XBRL taxonomy <https://xbrlview.ferc.gov/>`__ for the table in question.
+
+    The ``row_literals`` may contain elements of the FERC 1 form such as
+    headers that don't map to an XBRL column. These can be marked as ``headers`` in the
+    ``row_type`` column. Other values are either marked as ``report_value`` or
+    ``calculated_value`` to indicate whether the XBRL data is original or can be
```

Comment:
I thought that we only know whether a value is calculated or reported based on what it says in the XBRL taxonomy, hence the XBRL vs. DBF reference here.

Reply:
that is what I thought - which is why i think this column is silly and I still think we should delete it... but during some pr @zaneselvans indicated that is is supposed to be a dbf thing. even though it is almost always the same as what is in the xbrl metadata.

Reply:
hmmmm I feel like I used it in the context of XBRL when I was filling out the sheet.

Reply:
Ok here is an excerpt from #2040 that says: 

> indicates what kind of value is being reported in the DBF table in association with that row number.

I guess this makes sense because `header` really only appears in the DBF data.

Reply:
I updated the language here to say DBF instead and added some more clarity based on the detail from the above issue.